Sources/Clappr_iOS/Classes/Plugin/Base/DoubleTapPlugin.swift

```diff
@@ -15,11 +17,36 @@ public class DoubleTapPlugin: UICorePlugin {
     required init(context: UIObject) {
         super.init(context: context)
         animatonHandler = DoubleTapAnimation(core)
+        bindEvents()
     }
     
     required init() {
         super.init()
     }
+  
+    private func bindEvents() {
+        stopListening()
+        bindCoreEvents()
+    }
+    
+    private func bindCoreEvents() {
+        guard let core = core else { return }
+        listenTo(core, eventName: Event.didChangeActiveContainer.rawValue) { [weak self] _ in self?.bindEvents() }
+        listenTo(core, eventName: Event.didShowModal.rawValue) { [weak self] _ in self?.removeDoubleTapGesture() }
+        listenTo(core, eventName: Event.didHideModal.rawValue) { [weak self] _ in self?.addDoubleTapGesture() }
+    }
+    
+    func removeDoubleTapGesture() {
+        Logger.logInfo(#function)
```

Comment:
I got confused with this one. Is this supposed to be a `log` or should it remove the behaviour when a modal is open?